src/index.tsx

```diff
@@ -7,5 +7,5 @@ export default DocViewer;
 export { DocViewerRenderers } from "./renderers";
 export * from "./models";
 export * from "./utils/fileLoaders";
-export { type AvailableLanguages } from "./i18n";
+export { type AvailableLanguages, locales } from "./i18n";
```

Comment:
Ok but with this, we export json files...
We should export smth like `Object.keys()` of this `locales` object.

Reply:
Point taken. I have refactored the PR accordingly.